yarn.lock

```diff
@@ -1657,7 +1657,7 @@
   resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-2.0.172-rc.8.tgz#5daa62d1b11df87cdf26cbfd5e3aa7328e367e74"
   integrity sha512-vXhOcnlsYA/XgiaiPo4Cyp5GeI+CZE2zRVBNLkygemrafKnboLwUh9dLb/qC4YLsQ4DTaYPrvFq4cKujXBhl4w==
 
-"@griffel/babel-preset@1.3.2", "@griffel/babel-preset@^1.3.2":
+"@griffel/babel-preset@^1.3.2":
```

Comment:
This went wild, it seems that something was not bumped properly

Reply:
good catch 👍 I deduped, and now the only version is 1.3.3